### ISSUE TITLE
set index <true|false> like CLI parameter

### ISSUE THREAD

**Issue by PeHobt** [SIPp/sipp#411]
Hi all,

I'm little bit novice in using SIPp and so be patient with my questions ;-)

Speaking about "set index <true|false>" interactive command which is described [https://sipp.readthedocs.io/en/latest/controlling.html](url)

I'm able to use it from console, it seems to work fine. 

However it would be also useful to start this index displaying also from command line of the SIPp using some new parameter (e.g. -display_index).

I suppose such parameter isn't present in last version. And it would be useful in my opinion because SIPp frequently reports index in error messages and especially for long scenarios it is a little bit unpleasant to search which index is related to the trouble in given scenario.

What do you think about?

Btw: I made the local modification in my local copy of SIPp and now I have the following scenario screen

```
                                 Messages  Retrans   Timeout   Unexpected-Msg
0 :    REGISTER ---------->         1         0         0
1 :         401 <----------         1         0         0         0 
2 :    REGISTER ---------->         1         0         0                  
3 :         100 <----------         0         0         0         0        
4 :         200 <----------         1         0         0         0        
5 :      INVITE ---------->         1         0         0                  
6 :         401 <----------         1         0         0         0        
7 :         ACK ---------->         1         0                            
8 :      INVITE ---------->         1         0         0                  
9 :         100 <----------         1         0         0         0        
10:         200 <----------         1         0         0         0 
```

**Comment by wdoekes** [comment]
I like that :+1: 

I don't even see why we would have that option. Just enable the index always.

**Comment by PeHobt** [comment]
Hi, I'm glad that my suggestion is useful.

Now the question is if to add another parameter or make this modification permanent ;-)

I would vote for permanent modification without parameter because I think this feature is useful in any case and doesn't affect the basic screen too much.

My modification to make the modification permanent (without parameter) for 3.5.2 is

```
diff -r 7cef3db11427 -r 5e7394056918 src/logger.cpp
--- a/src/logger.cpp	Thu Nov 08 12:42:17 2018 +0100
+++ b/src/logger.cpp	Thu Dec 06 11:37:44 2018 +0100
@@ -55,7 +55,7 @@
 extern void print_stats_in_file(FILE * f);
 
 bool do_hide = true;
-bool show_index = false;
+bool show_index = true;
 
 void print_header_line(FILE *f, const char* pTestName)
 { 
```

For new parameter I cannot show since I have just permanent change ;-)

And I don't know what should be the next steps. Will you do the modification by yourself or do you expect to make it by me? (But maybe I don't have rights for this ;-), I have almost no experiences with GitHub work :-(, so I'm learning now...

**Comment by rkday** [comment]
Thanks! I've done this in #439 - I've also simplified things by removing the interactive command.

For future reference, http://jarednielsen.com/learn-git-fork-pull-request/ explains how to make and submit this sort of change.

**Comment by wdoekes** [comment]
Should be fixed by #439
